src/deepsparse/benchmark/benchmark_sweep.py

```diff
@@ -243,11 +243,13 @@ def _get_models(
     all_model_paths = model_paths.split(",")
     for model in all_model_paths:
         model = model.strip()
+        if not model:
+            continue
 
         if Path(model).is_file() or model.startswith("zoo:"):
             models.append(model)
         elif Path(model).is_dir():
-            models.extend([str(model) for model in Path.rglob("*.onnx")])
+            models.extend([str(model) for model in Path.rglob("*") if model.is_file()])
```

Comment:
we won't be able to support this glob pattern here - there are model formats which require multiple files (thinking specifically ONNX with external data exported here). we either need to have an approved list of extensions or just disable this feature in favor of explicitly providing path names

Reply:
On thinking more, removing this feature completely for getting this in